topics/javascript/index.md

```diff
@@ -8,6 +8,7 @@ short_description: JavaScript is an object-oriented programming language used al
   HTML and CSS to give functionality to web pages.
 topic: javascript
 url: https://developer.mozilla.org/en-US/docs/Web/JavaScript
+related: nodejs
 wikipedia_url: https://en.wikipedia.org/wiki/JavaScript
 ---
-JavaScript (JS) is a popular programming language which can be run in both backend and frontend contexts. Node.js is a programming language running on the V8 JavaScript execution engine. JS also brings life to frontend webpages and webapps, allowing content to be intelligent and highly interactive.
+JavaScript (JS) is a popular programming language which can be run in both backend and frontend contexts. JS brings life to frontend webpages and webapps, allowing content to be intelligent and highly interactive.
```

Comment:
@sadiqevani moving your comment here:
> Hey @kenyonj, I agree with your changes, though the description does not encapsulate what "JavaScript" is, what if we put the following text:

> JavaScript (JS) is a lightweight interpreted or JIT-compiled programming language with first-class functions. While it is most well-known as the scripting language for Web pages, many non-browser environments also use it, such as Node.js, Apache CouchDB and Adobe Acrobat. JavaScript is a prototype-based, multi-paradigm, dynamic language, supporting object-oriented, imperative, and declarative (e.g. functional programming) styles. Read more about JavaScript.

Reply:
I think what you have suggested is great. I would just remove the last sentence: `Read more about JavaScript`, for obvious reasons.

Reply:
@kenyonj done.